tools/base/requirements.in

```diff
@@ -1,4 +1,4 @@
-# Last updated 2024-10-30
+# Last updated 2024-12-06
```

Comment:
My bad, I actually updated them last week, but forgot to update the date here.

Thank you for refreshing them.